### PR TITLE
EPMRPP-50938. Auto merge launches.

### PR DESCRIPTION
**Pull request by ElenaRomanchuk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,9 +22,10 @@
   },
   "dependencies": {
     "glob": "7.1.3",
+    "minimatch": "^3.0.4",
     "mocha": "7.0.1",
     "node-ipc": "9.1.1",
-    "reportportal-client": "^5.5.0"
+    "reportportal-client": "https://github.com/reportportal/client-javascript.git#master"
```
Comment (DiscoElevator) [inline package.json:28]: I think now you can target the released version

Comment (ElenaRomanchuk) [reply package.json:28]: It will be done in scope of other task.

File: README.md [hunk 2]
```diff
@@ -172,9 +173,37 @@ ReportPortal also provides the corresponding methods for setting status into the
 To use custom filename in cy.screenshot function you should [setup ReportRortal custom commands](#setup-reportportal-custom-commands).
 Default usage of Cypress screenshot function is supported without additional setup. 
 
-## Merge launches
+## Automatically merge launches
 
-By default Cypress create a separate run for each test file. This section describe how to merge all this launches into a single launch in the end of run.
+By default Cypress create a separate run for each test file. This section describe how to report test items of differents specs into the single launch.
```
Comment (AmsterGet) [inline README.md:178]: items of `different` specs

File: lib/utils.js [hunk 3]
```diff
@@ -179,6 +180,58 @@ const getHookStartObject = (hook) => {
   };
 };
 
+const DEFAULT_SPEC_CONFIG = {
```
Comment (AmsterGet) [inline lib/utils.js:183]: Should it be located in `constants`?

Comment (ElenaRomanchuk) [reply lib/utils.js]: Done.

File: README.md (hunk 2 above)
Comment (ElenaRomanchuk) [reply README.md]: Fixed

**Pull request merged**